frontend/src/pages/LoginSignup/LoginCard.tsx

```diff
@@ -114,6 +114,8 @@ const LoginCard: React.FC<LoginCardProps> = ({ accountType }) => {
             // Handle other error codes (401 unauthorized, etc)
             console.error('There was a problem with the fetch operation:', error);
         }
+        //SW ADDED
+        navigate('/events');//Please only put this in the success path
```

Comment:
This navigation is already on line 107

Reply:
That's true. I am sorry about this mistake.